qt/scientific_interfaces/test/ISISReflectometry/Runs/RunsPresenterTest.h

```diff
@@ -49,6 +52,7 @@ class RunsPresenterTest : public CxxTest::TestSuite {
         m_searcher(nullptr), m_pythonRunner(), m_runNotifier(nullptr),
         m_runsTable(m_instruments, m_thetaTolerance, ReductionJobs()),
         m_searchString("test search string"), m_searchResult("", "", "") {
+    Mantid::API::FrameworkManager::Instance();
```

Comment:
Just trying to understand what this does?

Reply:
It initialises the framework, which is needed to be able to create algorithms in the tests.